frontend/src/i18n/translation.json

```diff
@@ -196,7 +196,7 @@
     "es": "Editor de código"
   },
   "WORKSPACE$BROWSER_TAB_LABEL": {
-    "en": "Browser",
+    "en": "Browser (Experimental)",
```

Comment:
Want to try translating this via ChatGPT or google translate? I bet ChatGPT will give you JSON in one shot

Reply:
Updated

Reply:
@rbren